x/concentrated-liquidity/lp.go

```diff
@@ -56,6 +56,21 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		return sdk.Int{}, sdk.Int{}, sdk.Dec{}, types.InsufficientLiquidityCreatedError{Actual: actualAmount1, Minimum: amount1Min}
 	}
 
+	denom0 := pool.GetToken0()
+	denom1 := pool.GetToken1()
+
+	// send deposit amount from position owner to pool
+	if actualAmount0.IsPositive() && actualAmount1.IsPositive() {
```

Comment:
Is this a requirement to be implemented for testing swaps? If not, can we submit this in another PR to keep this one focused on swaps only and to avoid blocking it on this?

Some notes:
- I'm not sure why we have so many if /else. Should be possible to:

```
if amount 0 is not positive => error
if amount 1 is not positive => error

bank send denom0 with amount 0
bank send denom 1 with amount 1
```
- this logic should be tested, might also make sense to encapsulate it in a helper